### PR TITLE
create new rubric author page component

### PR DESCRIPTION
**Pull request by stevenao**

### CODE REVIEW

File: demo/d2l-rubric-author-page.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
 
-	<title>d2l-rubric-editor demo</title>
+	<title>d2l-rubric-author-page demo</title>
```
Comment (mdgbayly) [inline demo/d2l-rubric-author-page.html]: Should we have separate demo pages?

File: editor/d2l-rubric-author-page.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<link rel="import" href="./d2l-rubric-editor.html">
```
Comment (mdgbayly) [inline editor/d2l-rubric-author-page.html]: Naming is hard, but I'm not really keen on the name `d2l-rubric-author-page`.  Doesn't really speak to me.
Open to other suggestions.... :-)
@mdulat @emil-furniss @wongvincent Any suggestions?!

Comment (mdgbayly) [reply editor/d2l-rubric-author-page.html]: d2l-rubric-editor with hindsight was too generic ! I really want to call this new component `d2l-rubric-editor`.

Comment (mdulat) [reply editor/d2l-rubric-author-page.html]: Ya that's what I'm thinking.

Comment (mdgbayly) [reply editor/d2l-rubric-author-page.html]: it's like the thing we built allready is the `d2l-rubric-criteria-overall-levels-editor` ha-ha.

Comment (mdulat) [reply editor/d2l-rubric-author-page.html]: Yup. What about `d2l-rubric-editor-complete` for the everything-editor? Or `d2l-rubric-whole-enchilada-editor`.

Comment (stevenao) [reply editor/d2l-rubric-author-page.html]: The decision is: d2l-rubric-author-page -> d2l-rubric-editor and d2l-rurbic-editor -> d2l-rubric-structure-editor.

**Pull request merged**